docker/Dockerfile.ci_arm

```diff
@@ -50,8 +50,8 @@ COPY install/ubuntu_install_sccache.sh /install/ubuntu_install_sccache.sh
 RUN bash /install/ubuntu_install_sccache.sh
 ENV PATH /opt/sccache:$PATH
 
-COPY install/ubuntu_install_llvm.sh /install/ubuntu_install_llvm.sh
-RUN bash /install/ubuntu_install_llvm.sh
+COPY install/ubuntu_install_llvm_from_source.sh /install/ubuntu_install_llvm_from_source.sh
+RUN bash /install/ubuntu_install_llvm_from_source.sh 15.0.7
```

Comment:
Can we add a hash parameter in here to verify the source tarball? Similar to:
https://github.com/apache/tvm/blob/main/docker/install/ubuntu_install_zephyr_sdk.sh#L45-L48